doc/api/fs.md

```diff
@@ -5000,17 +5025,17 @@ changes:
 
 * `path` {string|Buffer|URL}
 * `options` {Object}
-  * `emfileWait` {integer} If an `EMFILE` error is encountered, Node.js will
-  retry the operation with a linear backoff of 1ms longer on each try until the
-  timeout duration passes this limit. This option is ignored if the `recursive`
-  option is not `true`. **Default:** `1000`.
-  * `maxBusyTries` {integer} If an `EBUSY`, `ENOTEMPTY`, or `EPERM` error is
-  encountered, Node.js will retry the operation with a linear backoff wait of
-  100ms longer on each try. This option represents the number of retries. This
-  option is ignored if the `recursive` option is not `true`. **Default:** `3`.
+  * `maxRetries` {integer} If an `EBUSY`, `EMFILE`, `ENOTEMPTY`, or `EPERM`
+  error is encountered, Node.js will retry the operation with a linear backoff
```

Comment:
How about supporting exponential backoff or allow to custom, which may react to the real world.

Reply:
I don't plan on adding that myself, and definitely not in this PR. But someone could open a follow up PR that passes in a user defined function for that.